Library/Homebrew/dev-cmd/dispatch-build-bottle.rb

```diff
@@ -21,8 +21,8 @@ def dispatch_build_bottle_args
              description: "Build timeout (in minutes, default: 60)."
       flag   "--issue=",
              description: "If specified, post a comment to this issue number if the job fails."
-      comma_array "--macos=",
-                  description: "Version(s) of macOS the bottle should be built for."
+      comma_array "--macos",
```

Comment:
The other args end with `=` here though. Does this change the usage of the command?

Reply:
The `=` only has meaning for `flag`, indicating to the parser that an argument is required. It has no meaning for `switch`, which never takes arguments, nor for `comma_array`, which always requires them (and [eats any `=` that follows](https://github.com/Homebrew/brew/blob/1ce6a38730eea13a885e0e7704951004ae214ec4/Library/Homebrew/cli/parser.rb#L200)). In retrospect it may have been better to use a more obvious convention, e.g. `flag_opt` / `flag_req` or similar.